cobc/cobc.c

```diff
@@ -3001,6 +3019,80 @@ remove_trailing_slash (char *data)
 	}
 }
 
+static int
+string_is_dash (const char *s)
+{
+	return (strcmp (s, COB_DASH) == 0);
+}
+
+static void
+add_depend_target (const char *s)
+{
+	if (!cb_depend_target) {
+		cb_depend_target = cobc_strdup (s);
+	} else {
+		/* multiple invocations add to the list */
+		const size_t orig_len	= strlen (cb_depend_target);
+		const size_t new_len	= strlen (s);
+		const size_t buff_len	= orig_len + 1 + new_len + 1;
+		cb_depend_target = cobc_realloc (cb_depend_target, buff_len);
+		memset (cb_depend_target + orig_len, ' ', 1);
+		memcpy (cb_depend_target + orig_len + 1, s, new_len);
+		memset (cb_depend_target + orig_len + 1 + new_len, 0, 1);
```

Comment:
As those memset calls use only 1, maybe it is better to just assign that position? But I won't _request_ a change here.